src/renderer/classes/Note.js

```diff
@@ -0,0 +1,61 @@
+import '../typedef';
+import { defaultNoteOn } from '../constants/defaults';
+import { pick } from 'lodash';
+
+const properties = ['time', 'pitch', 'power', 'timbre'];
+let id = 0;
```

Comment:
Although I understand your intention here and I, myself, do the same thing sometimes, it's just simply better to have a `noteFactory` function/class/generator to enclose this variable in a scope.
i.e., something like this:
```javascript
class NoteFactory {
  constructor(){
    this.id = 1
  }
  create() {
     const note = new Note(this.id);
     this.id++;
     return note;
  }

}
 
```